lib/ui/home/home_page.dart

```diff
@@ -0,0 +1,246 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+class HomePage extends StatelessWidget {
+  late GoogleMapController mapController;
+  final LatLng _center = const LatLng(4.602796, -74.065841);
+
+  HomePage({super.key});
+
+  void _onMapCreated(GoogleMapController controller) {
+    mapController = controller;
+    rootBundle.loadString('assets/map_style.txt').then((string) {
+      mapController.setMapStyle(string);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    Color colorB1 = Color(int.parse("0961AD", radix: 16) + 0xFF000000);
+    Color colorB2 = Color(int.parse("2597FA", radix: 16) + 0xFF000000);
+    Color colorB3 = Color(int.parse("40A6FF", radix: 16) + 0xFF000000);
+
+    Color colorY1 = Color(int.parse("B89865", radix: 16) + 0xFF000000);
```

Comment:
Please add these colors to [theme/theme_constants](https://github.com/ISIS3510-202320-Team13/Flutter/blob/main/lib/ui/theme/theme_constants.dart)

Reply:
Done